media_driver/linux/common/ddi/media_libva_util.cpp

```diff
@@ -208,6 +208,8 @@ VAStatus DdiMediaUtil_AllocateSurface(
                 !(mediaSurface->surfaceUsageHint & VA_SURFACE_ATTRIB_USAGE_HINT_VPP_WRITE))
             {
                 tileformat = I915_TILING_NONE;
+                alignedWidth = MOS_ALIGN_CEIL(width, 2);
+                alignedHeight = MOS_ALIGN_CEIL(width, 2);
```

Comment:
should this be `MOS_ALIGN_CEIL(height, 2)`?

Reply:
It should be :)